build.gradle

```diff
@@ -1,10 +1,22 @@
 apply plugin: 'application'
 apply plugin: 'java'
 apply plugin: 'jacoco'
-mainClassName = 'main.java.Simulation'
+mainClassName = 'me.hkgumbs.main.java.tictactoe.Simulation'
 repositories {
     mavenCentral()
 }
+sourceSets {
+    main {
+        java {
+            srcDirs = ["src/me/hkgumbs/main/java/tictactoe"]
```

Comment:
This is much better - it makes the package name unique.

To go one step further, a more common directory structure would be `src/main/java/me/hkgumbs/tictactoe` and `src/test/java/me/hkgumbs/tictactoe`, with `src/main/java` and `src/test/java` as roots of the classpath, so that the package would be named `me.hkgumbs.tictactoe` (without `main.java` in the package name). [This project](github.com/jline/jline2) is a pretty decent demonstration of the usual maven project structure.